demostore_automation/tests/frontend/test_reg_and_submit.py

```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
```

Comment:
Put this code outside the main frameowrk.
I suggest you create a new repo callit 'random_scripts' or 'python_practice', or something like that. then check-in this script in that new repo.